app/models/user.rb

```diff
@@ -201,7 +201,7 @@ def email_tld
   end
 
   def name_for_paper_trail
-    "[User] #{full_name}"
+    "[User] #{full_name} (id=#{id})"
```

Comment:
Qu'est-ce que tu penserais de ce format ?
```suggestion
    "[User #{id}] #{full_name}"
```

Reply:
J'y ai réfléchi, au début j'aimais le côté compact, et ensuite j'ai trouvé que visuellement ça gâchait le côté "tag" de `"[Agent]"`, donc pas convaincu ! :slightly_smiling_face: